src/routingComps/Newsfeed.js

```diff
@@ -102,8 +124,8 @@ export default class newsfeeds extends Component {
 		}
```

Comment:
create button project doesn't redirect correctly to create project page

Reply:
Solved.

Reply:
Where is it solved, line 123 still shows href as CreateProject, it should be create-project

Reply:
Gonna be solved when merging.